save-cloud-charts/save-cloud/templates/backend-deployment.yaml

```diff
@@ -29,6 +29,33 @@ spec:
             - name: database-secret
               mountPath: {{ .Values.backend.dbPasswordFile }}
           {{- include "spring-boot.management" .Values.backend | nindent 10 }}
+      initContainers:
+        - name: git-cloner
+          image: alpine/git
+          args:
+            - clone
+            - --single-branch
+            - --
```

Comment:
empty arg?

Reply:
Honestly I've just copied this one from an example, but `git clone` supports a double dash to separate all named arguments from repo url